core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -1564,22 +1568,29 @@ protected void bitcoinSerializeToStream(OutputStream stream, boolean useSegwit)
             }
         }
         // lock_time
-        uint32ToByteStreamLE(lockTime, stream);
+        uint32ToByteStreamLE(vLockTime.rawValue(), stream);
     }
 
     /**
-     * Transactions can have an associated lock time, specified either as a block height or in seconds since the
-     * UNIX epoch. A transaction is not allowed to be confirmed by miners until the lock time is reached, and
+     * Transactions can have an associated lock time, specified either as a block height or as a timestamp (in seconds
+     * since epoch). A transaction is not allowed to be confirmed by miners until the lock time is reached, and
      * since Bitcoin 0.8+ a transaction that did not end its lock period (non final) is considered to be non
      * standard and won't be relayed or included in the memory pool either.
+     * @return lock time, wrapped in a {@link LockTime}
      */
+    public LockTime getWrappedLockTime() {
```

Comment:
How about just `.lockTime()` -- like it would be in a Java `record`?

Reply:
Renamed.